### PR TITLE
Adding links and descriptions of reference content

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f1f35dc](https://github.com/OfficeDev/office-scripts-docs/commits/f1f35dcc335b6d4b1bb9291088a8b9724bf6a24f)_: 

:clock10: **Pending**: waiting for processors ([11](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 11 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f1f35dc](https://github.com/OfficeDev/office-scripts-docs/commits/f1f35dcc335b6d4b1bb9291088a8b9724bf6a24f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-RefLinks/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C12%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912122038033134-47%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=iAacm1QXBviysb%2BLJSuYFlCtcCBPtT4%2FJOIkIqrFu9A%3D&st=2019-12-12T20%3A34%3A09Z&se=2020-01-12T20%3A39%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/scripting-fundamentals.md [hunk 1]
```diff
@@ -114,6 +114,20 @@ Running this script on the worksheet with the previous table creates the followi
 
 ![A column chart showing quantities of three items from the previous sales record.](../images/chart-sample.png)
 
+### Further reading on the object model
+
+The [Office Scripts API reference documentation](/javascript/api/office-scripts/overview) is a comprehensive listing of the objects used in Office Scripts. There, you can use the table of contents to navigate to any class you'd like to learn more about. Here are some commonly viewed pages:
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/scripting-fundamentals.md]:
```suggestion
The [Office Scripts API reference documentation](/javascript/api/office-scripts/overview) is a comprehensive listing of the objects used in Office Scripts. There, you can use the table of contents to navigate to any class you'd like to learn more about. The following are several commonly viewed pages.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[753892b](https://github.com/OfficeDev/office-scripts-docs/commits/753892b7f256f60ec7230d57e418ad34b20333a8)_: 

:clock10: **Pending**: waiting for processors ([24](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 24 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[753892b](https://github.com/OfficeDev/office-scripts-docs/commits/753892b7f256f60ec7230d57e418ad34b20333a8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-RefLinks/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C12%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912122251257878-47%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=m7g8%2BHqAqoVTNZiUWDT2JI7kP2JsfSh5KxngrIDiCVw%3D&st=2019-12-12T22%3A47%3A30Z&se=2020-01-12T22%3A52%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**